packages/app/src/app/pages/Sandbox/Editor/Workspace/items/NotOwnedSandboxInfo/NotOwnedSandboxInfo.tsx

```diff
@@ -2,17 +2,22 @@ import React, { useState } from 'react';
 
 import Dependencies from '../../Dependencies';
 import Files from '../../Files';
-// @ts-ignore
-import Project from '../../Project/index.tsx';
+import Project from '../../Project';
 import WorkspaceItem from '../../WorkspaceItem';
+import TemplateInfo from '../../Project/Template';
+
+import { useStore } from 'app/store';
 
 export const NotOwnedSandboxInfo = () => {
+  const {
+    editor: { currentSandbox: sandbox },
```

Comment:
@SaraVieira Destructur `sandbox` a bit further so the return value is a bit clearer (see below)

```suggestion
    editor: { currentSandbox: { git: { username } = {} } },
```